ipsec_offload_plugin/ipsec_offload/ipsec_grpc_connect.cc

```diff
@@ -74,6 +74,7 @@ string address = "127.0.0.1:50052";
 static std::string spi_path = "/ipsec-offload/ipsec-spi/rx-spi";
 static std::unique_ptr< ::grpc::ClientReaderWriterInterface<
        ::gnmi::SubscribeRequest, ::gnmi::SubscribeResponse>> stream_reader_writer;
+::grpc::ClientContext ctx_p;
```

Comment:
Lets make it static.

Reply:
made it static as it is file specific variable.